spec/api-menu-spec.ts

```diff
@@ -927,18 +928,20 @@ describe('Menu module', function () {
       w.show();
     });
 
-    it('does not crash when rendering menu item with Super or meta accelerator', async () => {
-      const menu = Menu.buildFromTemplate([{
-        label: 'Test Super',
-        accelerator: 'Super+Ctrl+T'
-      }, {
-        label: 'Test Meta',
-        accelerator: 'Meta+Ctrl+T'
-      }]);
-      const menuWillClose = once(menu, 'menu-will-close');
-      menu.popup({ window: w });
-      menu.closePopup();
-      await menuWillClose;
+    it('does not crash when rendering menu item with single accelerator combinations', async () => {
+      const chunkSize = 10;
+      for (let i = 0; i < singleModifierCombinations.length; i += chunkSize) {
+        const chunk = singleModifierCombinations.slice(i, i + chunkSize);
+        const menu = Menu.buildFromTemplate([
+          ...chunk.map(combination => ({
+            label: `Test ${combination}`,
+            accelerator: combination
+          }))
+        ]);
+        menu.popup({ window: w });
+        menu.closePopup();
+        await setTimeout(100);
```

Comment:
What caused the event to be replaced with a timer ?

Reply:
it hangs from the first call - i'm trying to figure out a better approach!

Reply:
It would be nice to keep the close event, sounds good on adding a timer to delay the next iteration.